### DISCUSSION TITLE
Some love to Blog Authors

### DISCUSSION THREAD

**Discussion by HitArrowLegend** [wintercms/wn-blog-plugin#11] [Ideas]
Hello, 

I think the Blog plugin should have an author section in which we could give them some love. The current way of doing things is nice when you only have one author because you can hardcode your way through creating an author block, but as soon as things become more crowded in the backend section, it's a different story.

Some ideas :
* The author section would be similar to the ones for categories and posts.
* Some of the really needed fields could be : First name, Last name, Nickname, Social medias, titles (like publisher, writer, etc...) 
* The author block should be a component.
* Blog Posts could also be filtered by authors

**Comment by mjauvin** [comment]
You can create backend users which will be used for Blog authors. You can also extend the backend user model in a plugin to have extra fields (like title, etc). That's what I do for my websites.

**Reply by HitArrowLegend** [reply]
I know that we can always have a plugin for everything, but authors are at the root of every blog posts, thats why I think an author block, even in its most basic form, should be added in core.
